tools/inspect/endline_whitespace_check.cpp

```diff
@@ -81,7 +84,7 @@ namespace boost
                     extend = 1;
                 }
                 size_t size = someline[p].size();
-                if (found < size - extend)
+                if (found < size - extend || found == someline[p].npos)
```

Comment:
FWIW, this could be written as `|| found == std::string::npos` (`npos` is a static member of std::string).